### PR TITLE
Replaced deploy-beta to main-action in deploy workflows

### PR DESCRIPTION
**Pull request by GPaoloni**
## Description
This PR moves the remaining deploy workflows from deploy-beta to use main-action instead. This means that the custom actions process is incorporated, but not running any extra step since there aren't any defined for this workflows (yet).

### Checklist
- [x] Corresponding issue has been opened https://bugs.benetech.org/browse/CHI-966
- [ ] N/A New tests added
- [ ] N/A Strings are localized

### CODE REVIEW

**Review by stephenhand — APPROVED** [review]

**Review by murilovmachado — APPROVED** [review]
LGTM! Can it be merged, already?

**Comment by GPaoloni** [comment]
@murilovmachado Yes, I was just waiting for a second review, as this is a kinda a dangerous change. Thanks for reviewing!

**Pull request merged**
